src/types/components/step-context/step-context.types.ts

```diff
@@ -0,0 +1,7 @@
+export type useStepContextType = () => {
+  handleStepData: (label: string, data: string) => void
+  stepData: {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [key: string]: any
+  }
```

Comment:
Have you used `stepData` in you code? If no just delete it for now, do not create `any` types.